.travis.yml

```diff
@@ -16,7 +16,8 @@ cache:
 
 before_script:
   - set -e
-  - export TAG=`if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi`
+  - if [ "$TRAVIS_BRANCH" == "latest" ]; then echo "Building branches named latest not allowed. Please rename your branch." && exit 1; fi;
```

Comment:
```suggestion
  - if [ "$TRAVIS_BRANCH" == "latest" ]; then echo "Building branches named latest not allowed as it would overwrite the stable docker image. Please rename your branch." && exit 1; fi;
```